### PR TITLE
Lung cancer vivarium modeling strategy

### PR DESCRIPTION
**Pull request by alibow**
This is currently in the GBD 2017 folder, although I think it might fit better in the GBD 2019 folder

### CODE REVIEW

File: docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst [hunk 1]
```diff
@@ -111,60 +118,53 @@ on the ages and sexes to which the cause applies.
...
-.. todo::
+Scope
++++++
+
+This Vivarium modeling strategy is intended to simulate TBL cancer incidence/morbidity as well as mortality so that it reflects the estimates and assumptions of GBD.
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+This model will assume the existence of a "recovered" cause model state in an attempt to be consistent with the GBD assumption that no morbidity due to TBL cancer does not occur more than ten years past incidence. Notably, since GBD performs the TBL cancer fatal and non-fatal models separately, this assumption in GBD applies to the non-fatal model only and not the fatal model, which is a limitation of our strategy. The assumption also asserts that there is no recurrance of TBL cancer.
```
Comment (aflaxman) [inline docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst:129]: "no morbidity does not occur" - double negative that I think means something different than you intended

**Review by aflaxman — APPROVED** [review]

File: docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst [hunk 2]
```diff
@@ -174,31 +174,54 @@ initialize the model.
...
-"""""""""""""""
-
-This table contains the measures needed for transition from one state to other in the cause model. 
+   * - I
+     - excess mortality rate
+     - :math:`\text{csmr_c426}\times\text{prevalence_c426}`
+     - 
+   * - I
+     - disabilty weights
+     - :math:`\displaystyle{\sum_{s\in\text{s_c426}}}\scriptstyle{\text{disability_weight}_s\,\times\,\text{prev}_s}`
```
Comment (yongqx) [inline docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst:207]: I think what software engineer needs is a weighted average across all sequelae within c426, we might want to add SUM(prev_s) as denominator.

File: docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst [hunk 3]
```diff
@@ -233,42 +251,26 @@ This table contains the data sources for all the measures. The table structure a
...
-     - 
-   * - disability_weight_s{`sid`}
-     - 
-     - 
-     - 
-   * - prevalence_s{`sid`}
-     - 
-     - 
-     - 
+   * - prevalence_c426
```
Comment (yongqx) [inline docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst:254]: In order to transform and load forecast data into vivarium data artifact, can we write down what metric Zach produced for lung cancer forecasts including prevalence, incidence, and deaths.

**Review by yongqx — APPROVED** [review]

File: docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst (hunk 3 above)
Comment (alibow) [reply docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst:254]: Can you clarify what you mean by this?

Comment (yongqx) [reply docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst:254]: The measure metric of forecast data provided by Zach is not always the GBD standard rate space. We'd better note the SEs what unit he produced and clarify the necessary transformation if needed. for example the original all-cause mortality rate forecast has a metric of log(acmr) per person-year.

**Pull request merged**